lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2305,7 +2305,118 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      ho_algorithm_config:
+        $ref: '#/definitions/ho_algorithm_configuration'
 
+  ho_algorithm_configuration:
+    description: ho algorithm configuration for enodeb configuration.
+    type: object
+    properties:
+      a1_threshold_rsrp:
+        type: integer
+        format: uint32
+        example: 50
+      lte_a1_threshold_rsrq:
+        type: integer
+        format: int32
+        example: 2
+      hysteresis:
+        type: integer
+        format: int32
+        example: 3
+      time_to_trigger:
+        type: string
+        example: "ms480"
+      a2_threshold_rsrp:
+        type: integer
+        format: int32
+        example: 5
+      lte_a2_threshold_rsrq:
+        type: integer
+        format: int32
+        example: 6
+      lte_a2_threshold_rsrp_irat_volte:
```

Comment:
All of these fields should have some more detailed description, value limits (min, max, or enum, etc.) as well as references to specifications to make the definitions easy to understand, and unambiguous. Many of these fields can also be further sub-grouped.

Reply:
![1640509171458_A45D7BBA-F021-4257-A860-7EA46E2D2A78](https://user-images.githubusercontent.com/87301024/147403941-c5916e1f-b092-4c1e-b4cb-817420bdc5d8.png)
Modified, thank you!